config/hardening/ssg-supplemental.sh

```diff
@@ -294,52 +292,58 @@ cat <<EOF > /etc/audit/rules.d/audit.rules
 -w /var/log/btmp -p wa -k session
 -w /var/log/wtmp -p wa -k session
 
+#2.6.2.4.12 Ensure auditd Collects System Administrator Actions
+-w /etc/sudoers -p wa -k actions
+
+#2.6.2.4.13 Make the auditd Configuration Immutable
+-w /sbin/insmod -p x -k modules
+-w /sbin/rmmod -p x -k modules
+-w /sbin/modprobe -p x -k modules
+-a always,exit -F arch=b32 -S init_module -S delete_module -k modules
+-a always,exit -F arch=b64 -S init_module -S delete_module -k modules
+
+# Ignore all the anonymous events. Often tagged on every rule, but ignoring 
```

Comment:
I can't take credit, I copied it from someone else. I'm leaving my clarification comment in place for future reviewers, even though our comments "crossed", since I didn't anchor my original comment on the correct line to make it clear what I was talking about.